_events/sr2019/competition.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Competition
+date: 2019-04-06 9:00:00
+layout: event
+type: competition
+location: Southampton University Highfield Campus
+---
+
+## Tickets
+
+Competitors need to bring along a ticket to be allowed entry. Anyone under the
+age of 18 must be accompanied by an adult. Student Robotics will announce when
+tickets are available.
+
+## Directions
+
+* How to [get to campus][soton-campus-directions]
+* [On-campus directions][soton-directions] (and parking)
```

Comment:
The linked document here is very specifically about Tech Days (and features that prominently within the document). I suggest we either drop the link (and reinstate it later when we have an updated document) or produce a new version of the document now.